### PR TITLE
RMET-225 - Update Cordova Camera Plugin to the last version released

### PR DESCRIPTION
**Pull request by CarlsCorrea**
## Description

Merge upstream (4.2.0) into OutSystems branch and keep the custom changes made by OutSystems
## Context

Closes the issue: https://outsystemsrd.atlassian.net/browse/RMET-225

Change required to keep SDK up to date
 
## Type of changes

- [ ] Feature (change which adds functionality)
    - [ ] BREAKING CHANGE (existing functionality will not work as expected due to new feature)
- [ ] Fix (change which fixes an issue)
    - [ ] BREAKING CHANGE (existing functionality will not work as expected due to fix)
- [x] Performance (change which improves performance)
    - [ ] BREAKING CHANGE (existing functionality will not work as expected due to performance improvement)
- [ ] Refactor (non-breaking change that is neither feature, fix nor performance)
- [ ] Style (non-breaking change that only affects formatting and/or white-space)

## Components affected
- [x] Android platform
- [x] iOS platform
- [x] JavaScript
- [ ] OutSystems

## Tests

Executed the P0s tests of the test plan

## Checklist

- [x] Pull request title follows the format `RNMT-XXXX <title>`
- [ ] Tests have been created
- [x] Code follows code style of this project
- [ ] CHANGELOG.md file is correctly updated
- [ ] Changes require an update to the documentation
	- [ ] Documentation has been updated accordingly

### CODE REVIEW

File: tests/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "cordova-plugin-camera-tests",
-  "version": "2.4.1-dev",
+  "version": "4.2.0",
```
Comment (GFOutsystems) [inline tests/package.json]: I believe this needs to be updated to 4.2.0-OS too

Comment (CarlsCorrea) [reply tests/package.json]: yes I will change.. needs to be OS... thanks

**Review by GFOutsystems — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,6 +1,5 @@
 <!--
```
Comment (vmfo22) [inline .github/PULL_REQUEST_TEMPLATE.md]: just wondering if we shouldn't use our PR template here.

**Review by vmfo22 — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (CarlsCorrea) [reply .github/PULL_REQUEST_TEMPLATE.md]: ok..  I will change it, because if I keep this it can confuse someone in the future

**Pull request merged**